docs/en_US/CommunitySharings/HpoComparision.md

```diff
@@ -5,17 +5,17 @@ Comparison of Hyperparameter Optimization algorithms on several problems.
 
 Hyperparameter Optimization algorithms are list below:
 
-- [Random Search](../Builtin_Tuner.md#Random)
-- [Grid Search](../Builtin_Tuner.md#Random)
-- [Evolution](../Builtin_Tuner.md#Evolution)
-- [Anneal](../Builtin_Tuner.md#Anneal)
-- [Metis](../Builtin_Tuner.md#MetisTuner)
-- [TPE](../Builtin_Tuner.md#TPE)
-- [SMAC](../Builtin_Tuner.md#SMAC)
-- [HyperBand](../Builtin_Tuner.md#Hyperband)
-- [BOHB](../Builtin_Tuner.md#BOHB)
-
-All algorithms run in NNI local environment。
+- [Random Search](../BuiltinTuner.md)
+- [Grid Search](../BuiltinTuner.md)
+- [Evolution](../BuiltinTuner.md)
+- [Anneal](../BuiltinTuner.md)
+- [Metis](../BuiltinTuner.md)
+- [TPE](../BuiltinTuner.md)
+- [SMAC](../BuiltinTuner.md)
+- [HyperBand](../BuiltinTuner.md)
+- [BOHB](../BuiltinTuner.md)
```

Comment:
please keep the previous version here.

Reply:
The previous version leads to 404.  The format **'xxx.md#position'** is not supported

Reply:
@suiguoxin if you change to “(../BuiltinTuner.md#BOHB)", it will work here.

Reply:
“(../BuiltinTuner.html#BOHB)" works.

Reply:
i have added a commit to fix it

Reply:
@suiguoxin Seems you should use ".md" instead of ".html"

Reply:
@xuehui1991  “(../xxx.md#position)" doesn't work since it will not be translated into 'xxx.html#position' automatically. So here we need to use ".html" directly.

Reply:
@suiguoxin  Can you open it?    [https://github.com/microsoft/nni/blob/master/docs/en_US/BuiltinTuner.md#Random](url)

Reply:
@suiguoxin Can you open this link? [https://github.com/microsoft/nni/blob/master/docs/en_US/BuiltinTuner.md#Random](url)

Reply:
delete #section from cross-file links to make links work in both readthedocs and github docs @xuehui1991 @QuanluZhang